pkg/policy/l3.go

```diff
@@ -133,6 +134,23 @@ func (m L3PolicyMap) DeepCopy() L3PolicyMap {
 	return cpy
 }
 
+func ipnetEqual(a, b net.IPNet) bool {
+	return bytes.Equal(a.IP, b.IP) && bytes.Equal(a.Mask, b.Mask)
```

Comment:
net.IP has the Equal method. `return a.IP.Equal(b.IP) && ...`

Reply:
Documentation states: "An IPv4 address and that same address in IPv6 form are considered to be equal." I though that might not be desirable and decided to err on the safe side; Deeming policies in-equal cases recalculation and while inefficient, calculating the same policy again is not breaking anything functionally.